index.bs

```diff
@@ -1875,8 +1875,14 @@ To <dfn>check if cookie-based debugging is allowed</dfn> given a
 
 <h3 algorithm id="obtaining-context-origin">Obtaining context origin</h3>
 
-To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|, return |node|'s [=node navigable=]'s
-[=navigable/top-level traversable=]'s [=navigable/active document=]'s [=origin=].
+To obtain the <dfn export for=node>context origin</dfn> of a [=node=] |node|:
+
+1. Optionally, return the embedded origin if it exists.
```

Comment:
Does `optionally` here mean that different implementations can choose whether they care about embedded origins and still be compliant with the specification?

Reply:
Yes, that is the intention to leave it defined by the user agent.

Reply:
I think we should clarify that. Otherwise, it could be interpreted as "some of the time, do this," when it really means that the implementation can choose whether it supports the notion of embedded origins, but if it does, this algorithm should be consistent.

Reply:
It could also apply to the case "some of the time, do this". For example, if we only want to do this for sources but not triggers.

Reply:
Right, but the point is that the algorithm should be consistent under particular static circumstances. For example, it is not in the spirit of this specification for the implementation to generate a random number to decide whether to return the embedded origin or actual origin.